### ISSUE TITLE
wysihtml5 image src and href are stripped

### ISSUE THREAD

**Issue by deanperry** [denkGroot/Spina#31]
See this Stackoverflow post - http://stackoverflow.com/questions/11773820/wysihtml5-image-src-and-href-are-stripped

It seems wysihtml5 strips the image src and href when local urls are used. Seeing as wysihtml5 isn't supported any more, should Spina move over to another wysiwyg framework?

**Comment by Bramjetten** [comment]
I'm actually preferring Markdown even. We've done some experiments with clients and it seems not everyone is ready for it yet though. Greatly depends on the kinds of layouts you're building.

**Comment by stefanvermaas** [comment]
@Bramjetten So images in text won't be supported? Or will Spina use a patched version of wysihtml5? Currently the implementation is broken.

**Comment by deanperry** [comment]
They do work if you supply an absolute URL, i.e an S3 URL but local doesn't work

**Comment by Bramjetten** [comment]
We upload everything to S3 so it works fine for us right now.

**Comment by stefanvermaas** [comment]
Ah okay! Is there any documentation about the S3 usage? Or should it be configured just like carrierwave?

**Comment by deanperry** [comment]
I added support with it #32 but you'll need to add the fog gem to your gemfile

**Comment by idl3** [comment]
How about use http://imperavi.com/redactor/ :)

**Comment by Bramjetten** [comment]
Redactor looks pretty cool at first sight. API is promising. What about the license though? How would that work?

**Comment by idl3** [comment]
I think if I purchased an OEM license, I can use it for any number of open source projects. But we will just be including it as a library and using it.

```
The OEM License holders are allowed to integrate the Software with open source products
```

**Comment by idl3** [comment]
@Bramjetten If you're good with Redactor I can purchase the license :ok_hand:

**Comment by Bramjetten** [comment]
I'm good with Redactor. Perhaps we should implement it in such a way that it's easily interchangeable.

**Comment by teanoon** [comment]
https://github.com/denkGroot/Spina-template/blob/master/vendor/assets/javascripts/spina/wysihtml5_parser_rules.js#L188
try this workaround: change `"src": "url"` to `"src": "src"`

**Comment by brodock** [comment]
There is also this new editor: http://trix-editor.org/ from basecamp folks

**Comment by Bramjetten** [comment]
We're actually looking into Trix and plan on integrating it in Spina. :)

**Comment by brodock** [comment]
awesome news :) :+1:

**Comment by Bramjetten** [comment]
Trix is implemented since 0.7.0.
